tests/ttnn/unit_tests/operations/ccl/perf/perf_csv.py

```diff
@@ -52,70 +57,147 @@ def clean_attributes(attributes):
     filtered_df["Input Shape"] = filtered_df.apply(
         lambda row: f"[{row['INPUT_0_W']}, {row['INPUT_0_Z']}, {row['INPUT_0_Y']}, {row['INPUT_0_X']}]", axis=1
     )
+    filtered_df["Output Shape"] = filtered_df.apply(
+        lambda row: f"[{row['OUTPUT_0_W']}, {row['OUTPUT_0_Z']}, {row['OUTPUT_0_Y']}, {row['OUTPUT_0_X']}]", axis=1
+    )
     filtered_df["Cycles Count"] = filtered_df["DEVICE FW END CYCLE"] - filtered_df["DEVICE FW START CYCLE"]
 
     def split_attributes(attributes):
         attr_dict = {key: value for key, value in re.findall(r"'([^']+)':\s*'([^']+)'", attributes)}
         attr_dict["topology"] = attr_dict.get("topology", "").split("::")[-1]
+        attr_dict["n_chips"] = int(attr_dict.get("ring_size", 1))
```

Comment:
We should just fail if the `ring_size` key isn't found. Otherwise the information is unreliable and meaningless.